api/query/cache/index/filter.go

```diff
@@ -120,6 +126,15 @@ func (tnp TestNamePattern) Filter(t TestID) bool {
 	return strings.Contains(name, tnp.q.Pattern)
 }
 
+// Filter interprets a SubtestNamePattern as a filter function over TestIDs.
+func (tnp SubtestNamePattern) Filter(t TestID) bool {
+	_, subtest, err := tnp.tests.GetName(t)
+	if err != nil || subtest == nil {
+		return false
+	}
+	return strings.Contains(*subtest, tnp.q.Subtest)
```

Comment:
what if we do strings.Contains(strings.ToLower(*subtest), strings.ToLower(tnp.q.Subtest))? I noticed lots of subtest capitalized the first letter, e.g. https://subtest-dot-wptdashboard-staging.appspot.com/results/WebCryptoAPI/secure_context/crypto-subtle-secure-context-available.https.sub.html?label=master&label=experimental&aligned

Reply:
Done.